File: version_3/code-samples/ClientApi/Commands/HowTo/GetDatabaseNames.cs
using Raven.Client.Document;

namespace Raven.Documentation.CodeSamples.ClientApi.Commands.HowTo
{
	public class GetDatabaseNames
	{
		private interface IFoo
		{
			#region get_database_names_1
			string[] GetDatabaseNames(int pageSize, int start = 0);
			#endregion
		}

		public GetDatabaseNames()
		{
			using (var store = new DocumentStore())
			{
				#region get_database_names_2
				var databaseNames = store
					.DatabaseCommands
					.GetDatabaseNames(0, 10);
				#endregion
			}
		}
	}
}

File: Documentation/3.5/Samples/java/src/test/java/net/ravendb/clientapi/commands/howto/GetDatabaseNames.java
package net.ravendb.clientapi.commands.howto;

import net.ravendb.client.IDocumentStore;
import net.ravendb.client.document.DocumentStore;


public class GetDatabaseNames {

  @SuppressWarnings("unused")
  private interface IFoo {
    //region get_database_names_1
    public String[] getDatabaseNames(int pageSize);

    public String[] getDatabaseNames(int pageSize, int start);
    //endregion
  }

  @SuppressWarnings("unused")
  public GetDatabaseNames() throws Exception {
    try (IDocumentStore store = new DocumentStore()) {
      //region get_database_names_2
      String[] databaseNames =
        store.getDatabaseCommands().getGlobalAdmin().getDatabaseNames(10);
      //endregion
    }
  }
}
